src/sdk.ts

```diff
@@ -137,3 +137,7 @@ export async function start(
 export async function user(): Promise<any> {
   return await request.getUser()
 }
+
+export async function team(): Promise<any> {
+  return await request.getTeam()
```

Comment:
if you are not going to handle the error here, you should simply return that way the error handling is done on the function that uses this function. In this specific case if you are going to await here, then you should try/catch or else in case of an error, this would end up being a unhandled promise rejection